DemoApp/Screens/DemoAppTabBarController.swift

```diff
@@ -65,8 +83,53 @@ class DemoAppTabBarController: UITabBarController, CurrentChatUserControllerDele
 
         draftListVC.tabBarItem.title = "Drafts"
         draftListVC.tabBarItem.image = UIImage(systemName: "bubble.and.pencil")
+        
+        reminderListVC.tabBarItem.title = "Reminders"
+        reminderListVC.tabBarItem.image = UIImage(systemName: "bell")
 
-        viewControllers = [channelListVC, threadListVC, draftListVC]
+        // Only show reminders tab if the feature is enabled
+        if AppConfig.shared.demoAppConfig.isRemindersEnabled {
+            viewControllers = [channelListVC, threadListVC, draftListVC, reminderListVC]
+        } else {
+            viewControllers = [channelListVC, threadListVC, draftListVC]
+        }
+    }
+    
+    // MARK: - Events Controller Setup
+    
+    private func setupEventsController() {
+        // Get the ChatClient instance from the currentUserController
+        let client = currentUserController.client
+        
+        // Initialize the events controller
+        eventsController = client.eventsController()
+        
+        // Set this class as the delegate for events
+        eventsController.delegate = self
+    }
+    
+    // MARK: - EventsControllerDelegate
+    
+    func eventsController(_ controller: EventsController, didReceiveEvent event: Event) {
+        if AppConfig.shared.demoAppConfig.isRemindersEnabled,
+           let reminderDueEvent = event as? ReminderDueEvent {
+            handleReminderDueEvent(reminderDueEvent)
+        }
+    }
```

Comment:
I'm not sure if this is going to stay the same since the goal is for the backend to send a remote push notification.

Reply:
do you know when that would be ready?

Reply:
I've deleted this part of the codebase. Push should be working eventually, and we should only merge once that is working, but at the moment the backend is still analysing why we don't receive a remote push.